### DISCUSSION TITLE
How do I allow people to search for my site?

### DISCUSSION THREAD

**Discussion by AHumanIBelieve** [community/community#47348] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I use GitHub Pages to host my webcomic([https://ahumanibelieve.github.io](https://ahumanibelieve.github.io)), and I am finding it great! However, my webcomic doesn't show up when typing in the name ('Wizards and Potters'), or any of the strip titles, but does show up(after telling google 'yes I mean 'AHumanIBelieve') when I type in my username. Is there a way to make it appear when searching 'Wizards and Potters', or any of the strip titles?

(please explain how to do them like you are explaining to a three year old, because I am useless when it comes to javascript/advanced html)

**Comment by joneslee0918** [comment]
hi guy.
I am joneslee0918.
where do you typing the name in?

**Reply by AHumanIBelieve** [reply]
google search - on [google.com](https://google.com), literally type in 'Wizards and Potters'

**Comment by joneslee0918** [comment]
The reason that when you type in  'AHumanIBelieve', your webcomic searched, because of that included username in url(https://ahumanibelieve.github.io/),
Coding in general is almost impossible.

**Comment by AHumanIBelieve** [comment]
So there is no way for me to make it appear in google search using code? I've tried doing stuff in google search console, would that help?

**Reply by joneslee0918** [reply]
Yes, it is good way.
But sometimes it can take a week or more for a search engine to update search results.

**Comment by AHumanIBelieve** [comment]
Thanks!

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
